crates/turborepo-lib/src/commands/link.rs

```diff
@@ -0,0 +1,206 @@
+use std::{
+    fs,
+    fs::{File, OpenOptions},
+    io,
+    io::{BufRead, Write},
+    os::unix::fs::PermissionsExt,
+};
+
+use anyhow::{anyhow, Context, Result};
+use console::Style;
+use dialoguer::{theme::ColorfulTheme, Confirm, Select};
+use dirs_next::home_dir;
+
+use crate::{
+    client::{CachingStatus, Team, UserClient},
+    commands::CommandBase,
+    ui::{BOLD, CYAN, GREY, UNDERLINE},
+};
+
+enum SelectedTeam<'a> {
+    User,
+    Team(&'a Team),
+}
+
+pub async fn link(mut base: CommandBase, modify_gitignore: bool) -> Result<()> {
+    let homedir_path = home_dir().ok_or_else(|| anyhow!("could not find home directory."))?;
+    let homedir = homedir_path.to_string_lossy();
+    println!(">>> Remote Caching");
+    println!();
+    println!("  Remote Caching shares your cached Turborepo task outputs and logs across");
+    println!("  all your team’s Vercel projects. It also can share outputs");
+    println!("  with other services that enable Remote Caching, like CI/CD systems.");
+    println!("  This results in faster build times and deployments for your team.");
+    println!(
+        "  For more info, see {}",
+        UNDERLINE.apply_to("https://turbo.build/repo/docs/core-concepts/remote-caching")
```

Comment:
```suggestion
        base.ui.apply(UNDERLINE.apply_to("https://turbo.build/repo/docs/core-concepts/remote-caching"))
```

Reply:
Does color include underline? I explicitly didn't apply this because I wasn't sure if the color config should apply to underline